qiskit/circuit/library/n_local/n_local.py

```diff
@@ -78,134 +78,170 @@ def n_local(
     circuit). Each layer is repeated ``reps`` times, and by default a final rotation layer is
     appended.
 
-    For instance, a rotation block on 2 qubits and an entanglement block on 4 qubits using
```

Comment:
Could you keep the drawing? It shows the circuit on a more abstract level, compared to the figures below which only show concrete implementations that might be harder to understand at a first glance